src/main/reactjs/src/components/clerkTranslator/ClerkTranslatorListing.tsx

```diff
@@ -112,6 +127,20 @@ const ListingRow = ({
           ))}
         </div>
       </TableCell>
+      <TableCell>
+        <div className="columns gapped-s">
+          <Button
+            to={translatorDetailsURL(translator.id)}
+            component={Link}
```

Comment:
Toimisiko tässä href?

Reply:
Ei oikein. Nythän noi menee react-routerin kautta, ts. navigoidaan in-app ja redux-tila säilyy. href olisi oikea navigointi ja pitäisi ladata kääntäjät uudestaan.